packages/core/src/scss/components/checkout/customer/_customer.scss

```diff
@@ -91,3 +91,49 @@
     margin-right: spacing("single");
     width: 100%;
 }
+
+.wallet-buttons-container {
```

Comment:
Everything here is not final.
All of them could be rewritten in the next ticket.